### PR TITLE
refactor: improve fee estimation error handling

### PR DESCRIPTION
**Pull request by mrfelton**
## Description:

Improve cryptic error messages in the log when estimating fees.

## Motivation and Context:

If you connect to an older version of lnd and try to make an onchain payment there are several rather cryptic warnings in the log.

## How Has This Been Tested?

Connect to testnet1 node and try to make an onchain payment

## Screenshots (if appropriate):

**Before:**
image

**After:**
image

## Types of changes:

refactor

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have reviewed and updated the documentation accordingly.
- [x] I have read the _CONTRIBUTING_ document.
- [ ] I have added tests to cover my changes where needed.
- [x] All new and existing tests passed.
- [x] My commits have been squashed into a concise set of changes.

### CODE REVIEW

File: utils/api.js [hunk 1]
```diff
@@ -70,6 +70,7 @@ export function requestNodeScores(chain, network) {
  * @returns {{fastestFee:number, halfHourFee:number, hourFee:number}} Recommended fees
  */
 export function requestFees(options) {
+  mainLog.info('Fetching fees from bitcoinfees.earn.com')
```
Comment (korhaliv) [inline utils/api.js]: Maybe let's use `BASE_URL`  for future proofness? it sometimes happens that a constant get changed and logs remain outdated

File: utils/error.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const createError = (msg, code) => {
+  const error = new Error(msg)
+  if (error.code) {
```
Comment (korhaliv) [inline utils/error.js]: Did you mean just `code` ?

File: utils/api.js (hunk 1 above)
Comment (mrfelton) [reply utils/api.js]: update

File: utils/error.js (hunk 2 above)
Comment (mrfelton) [reply utils/error.js]: lol, good catch

**Pull request merged**